### DISCUSSION TITLE
ss-rust不支持在配置文件中指定dns端口吗？

### DISCUSSION THREAD

**Discussion by YOU-Y8** [shadowsocks/shadowsocks-rust#972] [Q&A]
在libev的配置文件中nameserver可以指定为127.0.0.1:1053这种带端口的形式，rust好像不能这么设置。

**Comment by zonyitoo** [comment]
I don't think so, it should works.

https://github.com/shadowsocks/shadowsocks-rust/blob/ff3590a830a84b4ee4f4b98623897487eed43196/crates/shadowsocks-service/src/config.rs#L1959-L1970
